src/store/TableDataStore.js

```diff
@@ -209,6 +209,11 @@ export class TableDataStore {
     }
   }
 
+  getDataIgnoringPagination() {
```

Comment:
Perfect, calling function <code>getCurrentDisplayData</code> is only correct soultion